sklearn/metrics/ranking.py

```diff
@@ -286,7 +305,9 @@ def roc_auc_score(y_true, y_score, average="macro", sample_weight=None,
 
     max_fpr : float > 0 and <= 1, optional
         If not ``None``, the standardized partial AUC [3]_ over the range
-        [0, max_fpr] is returned.
+        [0, max_fpr] is returned. If multiclass task, should be either
```

Comment:
If multiclass task => In a multiclass task? For a multiclass task?

Also

as AUC ROC partial computation currently not supported

"is" is missing